contracts/access/manager/AccessManager.sol

```diff
@@ -572,15 +572,13 @@ contract AccessManager is Context, Multicall, IAccessManager {
 
         uint48 minWhen = Time.timestamp() + setback;
 
-        if (when == 0) {
-            when = minWhen;
-        }
-
-        // If caller is not authorised, revert
-        if (!immediate && (setback == 0 || when < minWhen)) {
+        // if call is not authorized, or if requested timing is sooner than
+        if ((!immediate && setback == 0) || (when > 0 && when < minWhen)) {
             revert AccessManagerUnauthorizedCall(caller, target, bytes4(data[0:4]));
         }
 
+        uint48 timepoint = uint48(Math.max(when, minWhen)); // cast is safe: both inputs are uint48
```

Comment:
In the NatSpec we're stating that

> The special value zero will automatically set the earliest possible time.

As far as I can tell, now we're setting to `minWhen` even if `when` is not 0 but less than `minWhen`. We should rephrase to:

> Any `when` that's smaller than the minimum required will be set to the earliest possible execution schedule.

Reply:
If when is smaller than minWhen but is not 0, we revert

Reply:
Noted, I couldn't tell!